sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -115,14 +111,15 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
   }
 
   override fun onStop() {
-    mapObserver.removeOnCameraChangeListener(cameraChangeListener)
-    mapObserver.removeOnMapChangedListener(mapChangeListener)
+    this.nativeObserver.onStop()
+    this.nativeObserver.removeOnCameraChangeListener(this.onCameraChangedListener)
+    this.nativeObserver.removeOnStyleLoadingFinishedListener(this.onStyleLoadingFinishedListener)
     renderer.onStop()
     pluginRegistry.onStop()
   }
 
   override fun onDestroy() {
-    mapObserver.clearListeners()
+    this.nativeObserver.clearListeners()
```

Comment:
why explicitly using`this` in the class?

Reply:
introduced during the refactor, will roll back.